assets/app/scripts/app.js

```diff
@@ -122,6 +122,10 @@ angular
         templateUrl: 'views/browse/build-config.html',
         controller: 'BuildConfigController'
       })
+      .when('/project/:project/browse/builds/:buildconfig/edit', {
```

Comment:
how about `/project/:project/browse/builds/edit/:buildconfig` ?? I know its not 100% REST friendly but could serve for this purpose.